test-shrinker/pom.xml

```diff
@@ -65,7 +65,7 @@
       <plugin>
         <groupId>com.github.wvengen</groupId>
         <artifactId>proguard-maven-plugin</artifactId>
-        <version>2.6.1</version>
+        <version>2.7.0</version>
```

Comment:
Can probably now remove the `Note: ... https://github.com/wvengen/proguard-maven-plugin/issues/388` comment a few lines below, because that has been resolved in version 2.7.0.

Reply:
Would you mind sending a separate PR for that after this one has been merged?

Reply:
Ok, here it is: https://github.com/google/gson/pull/2835